draft-edm-protocol-greasing.md

```diff
@@ -115,11 +119,15 @@ protocol constraints. For instance, protocols that use 8-bit fields may find it
 too costly to dedicate many grease values, while 32-bit or 64-bit fields are
 likely to have no limitations.
 
+## Use Grease Values at Unpredictable Times
+
 It is recommended that senders use grease values at unpredictable times or
 sequence points during protocol interactions. This avoids receivers
 unintentionally ossifying on the occurrence of greasing in the temporal or
 spatial domain.
 
+## Defining and Registering Grease Value Ranges
```

Comment:
```suggestion
## Define and Register Grease Value Ranges
```
Use part of speech consistent with the other headings added

Reply:
I was thinking the same =)